README.md

```diff
@@ -242,3 +243,7 @@ your openstack Cluster API Kubernetes cluster.
 ### Trouble shooting
 
 Please refer to [Trouble shooting documentation](docs/trouble_shooting.md) for further info.
+
+## Development Guide
```

Comment:
Should we rename this to Contributing and also refering to CONTRIBUTING.md?

```
## Contributing

Please refer to the [Contributing Guide](CONTRIBUTING.md) and [Development Guide] (docs/development.md) for this project.
```

or something like this?

Reply:
OK, I will update from Development to  Contributing.

Reply:
> OK, I will update from Development to Contributing.

I was wrong. I understand your intent. I will fix this as your proposal.